src/util/iterative_restoration.jl

```diff
@@ -27,22 +27,79 @@ function run_iterative_restoration(network, model_constructor, optimizer; repair
     _PMs.propagate_topology_status!(network_mld)
 
     result_mld = run_mld_strg(network_mld, model_constructor, optimizer, kwargs...)
+
     clean_status!(result_mld["solution"])
 
     ## Turn network into multinetwork solution to merge with solution_iterative
     mn_network_mld = _PMs.replicate(result_mld["solution"], 1)
     mn_network_mld["nw"]["0"] = mn_network_mld["nw"]["1"]
-    delete!(mn_network_mld["nw"],"1")
+    delete!(mn_network_mld["nw"], "1")
     result_mld["solution"] = mn_network_mld
 
     Memento.info(_PMs._LOGGER, "begin Iterative Restoration")
     result_iterative = _run_iterative_sub_network(network, model_constructor, optimizer; repair_periods=repair_periods, kwargs...)
     merge_solution!(result_iterative, result_mld)
 
+    # this sets the status of components that were removed by
```

Comment:
Does this ensure that the devices removed from the MLD network are viewed in the final network?


If you take issue with non-variable device statuses in the result, I think it should be possible to do a 1-time period restoration, and take the network 0 result to use for reporting the solution

Reply:
Correct.  If the component was active in the initial network but was inactive by the end of the MLD solve, it will get an explicit `status = 0` in the solution.